### PR TITLE
Making changes to Get-All_Sections_and _Memberships.ps1 to have ExchangeOnlineManagement

### PR DESCRIPTION
**Pull request by mohankrishnaseelam**
Making changes to Get-All_Sections_and _Memberships.ps1 to have ExchangeOnlineManagement

### Related Issue
[_Please put the related issue number to reference if applicable_  ](https://dev.azure.com/1edu/DIP%20Master/_workitems/edit/132835)

### Summary
Modified to have ExchangeOnlineManagement module

### Reason
_Why is this change being proposed?_

### Proposed Change(s)
- change 1
- change 2
- change N

### Details
_Detailed description of the change(s)_

### Additional Info
_Any additional information or context_

### References
[Example](www.microsoft.com)

### Checklist
- [ ] Documentation

### CODE REVIEW

File: SDS Scripts/Get-All_Sections_and_Memberships.ps1 [hunk 1]
```diff
@@ -1,66 +1,105 @@
...
 
+try {
+    Import-Module ExchangeOnlineManagement | Out-Null
+}
+catch {
+    Write-Error "Failed to load Exchange Online Management Module"
+    Get-Help -Name .\Get-All_Sections_and_Memberships.ps1 -Full | Out-String | Write-Error
+    throw
+}
+
```
Comment (AyronJohnson) [inline SDS Scripts/Get-All_Sections_and_Memberships.ps1:98]: Add Connect-ExchangeOnline

File: SDS Scripts/Get-All_Sections_and_Memberships.ps1 [hunk 2]
```diff
@@ -1,66 +1,105 @@
...
+1. Install Exchange Online Management Module with command 'Install-Module ExchangeOnlineManagement'
+
+2. Check that you can connect to your tenant directory from the PowerShell module to make sure everything is set up correctly.
+
+    a. Open a separate PowerShell session
+    
+    b. Execute: "Connect-ExchangeOnline -Device" to bring up a sign-in UI.
+    The command returns a URL and unique code that's tied to the session. You need to open the URL in a browser on any computer, and then enter the unique code.
+    After you complete the login in the web browser, the session in the Powershell 7 window is authenticated via the regular Azure AD authentication flow, and the Exchange Online cmdlets are imported after few seconds.
```
Comment (AyronJohnson) [inline SDS Scripts/Get-All_Sections_and_Memberships.ps1:26]: -Device not necessary.  Remove PowerShell 7 and just leave PowerShell as this should be able to run in prior versions and hopefully some future ones.

File: SDS Scripts/Get-All_Sections_and_Memberships.ps1 (hunk 1 above)
Comment (mohankrishnaseelam) [reply SDS Scripts/Get-All_Sections_and_Memberships.ps1:99]: As suggested updated the code.

File: SDS Scripts/Get-All_Sections_and_Memberships.ps1 (hunk 2 above)
Comment (mohankrishnaseelam) [reply SDS Scripts/Get-All_Sections_and_Memberships.ps1]: As suggested updated the comment.

**Review by AyronJohnson — CHANGES_REQUESTED** [review]
Just noticed that the group owner column is listing System.Object.  I think this is because there are multiple owners.  Can we list multiple?

**Review by AyronJohnson — APPROVED** [review]

**Pull request merged**